Stack/Opc.Ua.Core/Schema/ApplicationConfiguration.cs

```diff
@@ -250,7 +251,7 @@ public bool DisableHiResClock
         private string m_sourceFilePath;
 
         private ServiceMessageContext m_messageContext;
-        private CertificateValidator m_certificateValidator;
```

Comment:
this should remain as it is
https://github.com/OPCFoundation/UA-.NETStandard/pull/1161/files#r509056300

Reply:
Removed the `readonly` modifier and put initialization back into the `Initialize` method.